config/action.d/allow-iptables-multiport.conf

```diff
@@ -0,0 +1,59 @@
+# Fail2Ban configuration file for allowing hosts
+#
+# WARNING
+# Please be aware that all users behind NAT will access the service on the specified port.
+# You should protect this service with another jail that has very long bantime.
+
+[INCLUDES]
+
+before = iptables-common.conf
+
+[Definition]
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = iptables -N f2b-<name>
+              iptables -A f2b-<name> -j <blocktype>
```

Comment:
> add "returntype" variable defaulting to "RETURN" in -commons

I don't dare to touch other filters.



> why not ACCEPT?

`# Notes: ACCEPT skips other chains`

It is necessary to be able to block "pass2allow"-ed IP-s.